typescript/phoenix-sdk/src/market.ts

```diff
@@ -1174,9 +1115,15 @@ export class Market {
     matchLimit?: number;
     clientOrderId?: number;
     useOnlyDepositedFunds?: boolean;
-    lastValidSlot?: number;
-    lastValidUnixTimestampInSeconds?: number;
+    lastValidSlot?: number | null;
+    lastValidUnixTimestampInSeconds?: number | null;
   }): OrderPacket {
+    if (lastValidSlot === undefined) {
+      lastValidSlot = null;
+    }
+    if (lastValidUnixTimestampInSeconds === undefined) {
+      lastValidUnixTimestampInSeconds = null;
+    }
```

Comment:
This is really ugly, do we need to do this

Reply:
The type coercion is kinda annoying. I would prefer to allow users to omit the fields that have type `number | null`. Do you know a better way to do this?

Reply:
we should probably just not allow null, wdyt?

Reply:
Yeah we can just make the input number | undefined and then coerce the undefined to nulls

Reply:
why do we need null anywhere? can't we just use undefined always?